components/brave_news/browser/resources/Feed.tsx

```diff
@@ -208,7 +208,7 @@ export default function Component({ feed, onViewCountChange, onSessionStart }: P
         {el}
       </div>
     })
-  }, [cardCount, feed?.items, registerViewDepthObservation])
+  }, [cardDisplayCount, feed?.items, registerViewDepthObservation])
```

Comment:
This is to avoid re-rendering when the `cardCount` is already at or beyond the feed item length.